pkg/controller/operandrequest/operandrequest_controller_test.go

```diff
@@ -58,6 +58,27 @@ func TestRequestController(t *testing.T) {
 	deleteOperandRequest(t, r, req, requestInstance)
 }
 
+// TestOperandConfig runs ReconcileOperandRequest.Reconcile() against a
+// fake client that checks operand reconcile function.
+func TestOperandConfig(t *testing.T) {
+	var (
+		name      = "common-service"
+		namespace = "ibm-common-service"
+	)
+
+	req := getReconcileRequest(name, namespace)
+	r := getReconciler(name, namespace)
+	requestInstance := &v1alpha1.OperandRequest{}
+
+	initReconcile(t, r, req, requestInstance)
+
+	absentOperandCustomResource(t, r, req, requestInstance)
+
+	presentOperandCustomResource(t, r, req, requestInstance)
```

Comment:
@horis233 I think we don't need a new func for custom resource test, you can add these 2 tests under https://github.com/IBM/operand-deployment-lifecycle-manager/blob/fa186944a8aad3709e1a8c029e09bcd79f5edadc/pkg/controller/operandrequest/operandrequest_controller_test.go#L56

Reply:
@DanielXLee Done